concrete/src/Entity/Notification/UserDeactivatedNotification.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Concrete\Core\Entity\Notification;
+
+use Concrete\Core\Notification\View\UserDeactivatedListView;
+use Concrete\Core\User\Event\DeactivateUser;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(
+ *     name="UserDeactivatedNotifications"
+ * )
+ */
+class UserDeactivatedNotification extends Notification
+{
+
+    /**
+     * The user that is being deactivated
+     *
+     * @ORM\Column(type="integer", options={"unsigned": true})
+     */
+    protected $userID;
+
+    /**
+     * The user doing the deactivating
+     *
+     * @ORM\Column(type="integer", options={"unsigned": true, "nullable": true})
+     */
+    protected $actorID = null;
```

Comment:
There's no need to initialize a variable to `null`: it's the default value.

Reply:
True, I won't declare = null in future PRs